www/front_src/src/components/ReactRouter/index.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 
 import { connect } from 'react-redux';
 import { Switch, Route, withRouter } from 'react-router-dom';
-import { isEmpty, equals } from 'ramda';
+import { equals } from 'ramda';
```

Comment:
`equals` is not used here

Reply:
And here ? :  line 102 to 106
 (previousProps, nextProps) =>
    equals(previousProps.pages, nextProps.pages) &&
    equals(previousProps.allowedPages, nextProps.allowedPages) &&
    equals(previousProps.externalPagesFetched, nextProps.externalPagesFetched),
);

I just want removed isEmpty, equals was present before my commit